### DISCUSSION TITLE
Turning on debug prints for pdngen?

### DISCUSSION THREAD

**Discussion by donn** [The-OpenROAD-Project/OpenROAD#5075] [Q&A] [answered]
`-verbose` appears to be deprecated for pdngen. We're running into a weird issue with a proprietary PDK where only one ring is getting generated. Is there a way to turn on debug prints…?

**Comment by maliberty** [comment, marked as answer]
There are lots of debug prints and in general you turn them on with set_debug_level (see https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/docs/contrib/Logger.md#debug-messages)
